src/sentry/usage_accountant/accountant.py

```diff
@@ -82,6 +84,6 @@ def record(
         )
 
         _accountant_backend = UsageAccumulator(producer=producer)
-        atexit.register(_shutdown)
+        register_shutdown(_shutdown)
```

Comment:
I am not sure this will be enough.
This method is not supposed to be celery specific. We need to be able to shutdown the accountant in other environment as well. 

I'd assume your goal is to replace the `atexit` shutdown with the patch you added here. But that is only going to work for celery.

Reply:
we can continue to register it on atexit then, but it remains to be tested whether that actually works on uwsgi